Code/Framework/AzFramework/AzFramework/Script/ScriptRemoteDebugging.cpp

```diff
@@ -418,6 +421,18 @@ namespace AzFramework
         {
             m_executionState = SDA_STATE_PAUSED;
 
+#if AZ_TRAIT_AZFRAMEWORK_SHOW_MOUSE_ON_LUA_BREAKPOINT
+            // We are about to block the main thread, only allowing to process
+            // network events. This works fine on Windows, but on Linux the mouse
+            // pointer doesn't show up when the user ALT+TAB out of the Editor window.
+            // We need to make mouse cursor visible again and it fixes all the Linux
+            // problems, and it doesn't hurt Windows either.
+            AzFramework::InputSystemCursorRequestBus::Event(
+                AzFramework::InputDeviceMouse::Id,
+                &AzFramework::InputSystemCursorRequests::SetSystemCursorState,
+                AzFramework::SystemCursorState::UnconstrainedAndVisible);
```

Comment:
```suggestion
            // Store the current system cursor state so we can restore it later
            auto systemCursorState = AzFramework::InputSystemCursorRequestBus::Event(
                AzFramework::InputDeviceMouse::Id,
                &AzFramework::InputSystemCursorRequests::GetSystemCursorState)
            
            AzFramework::InputSystemCursorRequestBus::Event(
                AzFramework::InputDeviceMouse::Id,
                &AzFramework::InputSystemCursorRequests::SetSystemCursorState,
                AzFramework::SystemCursorState::UnconstrainedAndVisible);
```

Reply:
@AMZN-alexpete Event buses can't return values

Reply:
Will make the suggested changes.

Reply:
Done.